awx/main/dispatch/worker/callback.py

```diff
@@ -102,6 +104,7 @@ def flush(self, force=False):
         now = tz_now()
         if (
             force or
+            (time.time() - self.last_flush) > settings.JOB_EVENT_BUFFER_SECONDS or
```

Comment:
*this* timeout, however, should be shorter and enforced as an *actual* "time that has passed since the last *flush*" check per-read

Reply:
@chrismeyersfsu I'm fairly confident that this fixes the use case outlined in the associated issue, but I want to try all of this together in a large-volume deployment on a large VM (to make sure we're not losing some of our performance gains in prior releases)